ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -469,41 +471,57 @@ def add_capacity(osd_size_capacity_requested):
         device_sets_required + old_storage_devices_sets_count
     )
     lvpresent = localstorage.check_local_volume()
-    if lvpresent:
-        ocp_obj = OCP(
-            kind="localvolume", namespace=config.ENV_DATA["local_storage_namespace"]
-        )
-        localvolume_data = ocp_obj.get(resource_name="local-block")
-        device_list = localvolume_data["spec"]["storageClassDevices"][0]["devicePaths"]
-        final_device_list = localstorage.get_new_device_paths(
-            device_sets_required, osd_size_capacity_requested
-        )
-        device_list.sort()
-        final_device_list.sort()
-        if device_list == final_device_list:
-            raise ResourceNotFoundError("No Extra device found")
-        param = f"""[{{ "op": "replace", "path": "/spec/storageClassDevices/0/devicePaths",
-                                                 "value": {final_device_list}}}]"""
-        log.info(f"Final device list : {final_device_list}")
-        lvcr = localstorage.get_local_volume_cr()
-        log.info("Patching Local Volume CR...")
-        lvcr.patch(
-            resource_name=lvcr.get()["items"][0]["metadata"]["name"],
-            params=param.strip("\n"),
+    ocp_version = get_ocp_version()
+    platform = config.ENV_DATA.get("platform", "").lower()
+    is_lso = config.DEPLOYMENT.get("local_storage")
+    if (
+        ocp_version == "4.7"
```

Comment:
Any reason for running this only on OCP 4.7? Add-capacity UI is same even for older OCP versions and I think also in 4.8

Reply:
I tested it on OCP4.7.
Maybe on different versions the elements(ui) are different